students/JamesTraub/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# James Traub 04-04-18
+# Lesson 3 - String Formatting Lab Exercise
+
+# Task 1
+tuple_elements = ( 2, 123.4567, 10000, 12345.67)
+print('file_{0:03d}: {1:.2f}, {2:.2e}, {3:.2e}'.format(tuple_elements[0], tuple_elements[1], tuple_elements[2], tuple_elements[3]))
+
+# Task 2
+te1 = tuple_elements[0]
+te2 = tuple_elements[1]
+te3 = tuple_elements[2]
+te4 = tuple_elements[3]
+print(f"file{te1:03d}: {te2:.2f}, {te3:.2e}, {te4:.2e}")
```

Comment:
you can also access elements directly in f-string: `f"file{tuple_elements[0]}"` which is more lengthy but more applicable in real scenarios